net/bulk_emulation/mem_xprt_pvt.h

```diff
@@ -36,9 +36,9 @@
 */
 
 enum {
-	M0_NET_BULK_MEM_MAX_BUFFER_SIZE     = (1 << 19),
-	M0_NET_BULK_MEM_MAX_SEGMENT_SIZE    = (1 << 18),
-	M0_NET_BULK_MEM_MAX_BUFFER_SEGMENTS = 256,
+	M0_NET_BULK_MEM_MAX_BUFFER_SIZE     = (1 << 20),
+	M0_NET_BULK_MEM_MAX_SEGMENT_SIZE    = (1 << 20),
+	M0_NET_BULK_MEM_MAX_BUFFER_SEGMENTS = 1,
```

Comment:
Please explain the reason of these changes.

Reply:
In case of libfabric, for RPC max seg_nr = 1 and max segment size = 1MB